index.html

```diff
@@ -5517,29 +5533,23 @@ <h2 id="AudioWorkletProcessor">
               </p>
             </dd>
             <dt>
-              attribute EventHandler onmessage
+              attribute EventHandler ondata
             </dt>
             <dd>
               <p>
-                The <code>onmessage</code> handler is called whenever the
-                <a><code>AudioWorkletNode</code></a> posts a message back to
-                the corresponding processor object.
+                The <code>ondata</code> handler is called whenever the
+                <a><code>AudioWorkletNode</code></a> posts data to the
+                corresponding processor object.
```

Comment:
I agree, but I think we should focus on the IDL/example change in this PR, then we can deal with the messaging part later.